### PR TITLE
Update README image URL

### PR DESCRIPTION
**Pull request by rgantzos**
### Changes

Replaces the README popup image with the scratchaddons.com image.

### Reason for changes

The old one looked old and outdated, especially because it highlighted a 2.0 page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Scratch Addons combines new and existing features and themes for the [Scratch](h
 
 <div align="center">
 
-![Scratch Addons screenshot](.github/readme-images/screenshot.png)
+![intro-hero](https://github.com/ScratchAddons/ScratchAddons/assets/86856959/3cd47267-df00-4f0c-babe-73b3e6c4e4f0)
```
Comment (DNin01) [inline README.md:22]: We should probably store the image in the repository, inside `.github/readme-images/`.

**Review by DNin01 — CHANGES_REQUESTED** [review]

**Comment by Samq64** [comment]
> The old one looked old and outdated, especially because it highlighted a 2.0 page.

That was kind of the point since I was trying to highlight Scratch 2 to Scratch 3.

File: README.md (hunk 1 above)
Comment (WorldLanguages) [reply README.md:22]: Also why remove the descriptive alt text?

**Comment by WorldLanguages** [comment]
I think we could keep both images in sync. Related to other issues I don't have the energy to search #for.

File: README.md (hunk 1 above)
Comment (Samq64) [reply README.md:22]: Why not use the version from `website-v2` so they stay in sync?

**Review by DNin01 — APPROVED** [review]
Fetching from scratchaddons.com works, and means we only have to update the image in one place.

**Review by Samq64 — APPROVED** [review]

**Comment by Samq64** [comment]
Actually, could the previous one be deleted?

**Comment by DNin01** [comment]
> Actually, could the previous one be deleted?

Done.

**Review by Hans5958 — APPROVED** [review]
LGTM

**Pull request merged**